tests/phpunit/wp-tests-config.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * WordPress configuration file for PHPUnit tests.
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+// Test with WordPress debug mode (default).
+define( 'WP_DEBUG', true );
+
+// Set path to WordPress installation.
+define( 'ABSPATH', dirname( dirname( __DIR__ ) ) . '/vendor/roots/wordpress/' );
+
+/*
+ * This configuration file will be used by the copy of WordPress being tested.
+ * wordpress/wp-config.php will be ignored.
+ *
+ * WARNING WARNING WARNING!
+ * These tests will DROP ALL TABLES in the database with the prefix named below.
+ * DO NOT use a production database or one that is shared with something else.
+ */
+
+define( 'DB_NAME', 'wordpress_test' );
+define( 'DB_USER', 'root' );
+define( 'DB_PASSWORD', 'example' );
+define( 'DB_HOST', '127.0.0.1:9306' );
+define( 'DB_CHARSET', 'utf8' );
+define( 'DB_COLLATE', '' );
+
+/**#@+
+ * Authentication Unique Keys and Salts.
+ *
+ * Change these to different unique phrases!
+ * You can generate these using the {@link https://api.wordpress.org/secret-key/1.1/salt/ WordPress.org secret-key service}
+ */
```

Comment:
This part isn't really relevant for this context as there's no point in generating these or using secure values since they aren't intended to be updated here.
```suggestion
```